smartseq2_single_sample/run_multi_samples/ss2_multi_sample_wf.wdl

```diff
@@ -0,0 +1,98 @@
+import "ss2_single_sample_wf.wdl" as singlesample
+
+task GatherMetrics {
+
+  Array[File]+ input_metrics_fn
+  String output_filename
+   
+  command <<<
+    # To parse out Picard metrics
```

Comment:
If I understand right, you're taking the header from the first file and then copying in the data rows from all files. Can you add a comment that says that? Alternatively, this might be clearer in Python.

Reply:
I originally tried to do it with python but had difficult to pass files in array structure.

Reply:
A formalism that works to convert an Array[File] into a python list(str) is:
```
python_list_name = ['{sep="', '" wdl_array_variable_name}']
```

Reply:
yea... this will work..... python_list_name = '${sep="," array_file_var}'.split(",")

Anyway, I took bash out and add python script in.